infrastructure-configs/demeter.json

```diff
@@ -2,9 +2,17 @@
 
 "substitutions": {
     "AOCS_HDFS_BAMS": "hdfs:///datasets/ovarian/ega-tim",
-    "HG19_FASTA": "/hpc/users/ahujaa01/reference_genomes/hg19-reference/ucsc.hg19.fasta"
+    "HG19_FASTA": "/hpc/users/ahujaa01/reference_genomes/hg19-reference/ucsc.hg19.fasta",
+    "SPARK_HOME": "/hpc/users/willir31/sparks/spark-1.6.1-bin-hadoop2.6"
 },
 
+"environment_variables": {
+    "SPARK_HOME": "$SPARK_HOME",
+    "YARN_CONF_DIR": "/etc/hadoop/conf"
+},
+
+"spark_submit": "$SPARK_HOME/bin/spark-submit",
+
```

Comment:
- `spark.executor.memory` is duplicated with `--executor-memory`
- `spark.storage.memoryFraction` is deprecated and is now `spark. memory. storageFraction` does something similar